lib/coinbase/webhook.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of a Webhook.
+  class Webhook
+    class << self
+      # Creates a new Webhook on the specified Network and generate a default address for it.
+      # @param network_id [String]  the ID of the blockchain network.
+      # @return [Coinbase::Webhook] the new Webhook
+      def create(network_id:, notification_uri:, event_type:, event_filters:)
+        model = Coinbase.call_api do
+          webhooks_api.create_webhook(
+            create_webhook_request: {
+              network_id: Coinbase.normalize_network(network_id),
+              notification_uri: notification_uri,
+              event_type: event_type,
+              event_filters: event_filters
+            }
+          )
+        end
+
+        new(model)
+      end
+
+      def list_webhooks_page(page)
+        webhooks_api.list_webhooks({ limit: DEFAULT_PAGE_LIMIT, page: page })
+      end
+
+      def update_webhook(webhook_id, network_id:, notification_uri:, event_type:, event_filters:)
```

Comment:
Both `update` and `delete` should be instance methods, where the `webhook` already contains the `webhook_id`!